client/web-sveltekit/src/routes/search/QueryExamples.svelte

```diff
@@ -2,20 +2,25 @@
     import type { Readable } from 'svelte/store'
 
     import { isDefined } from '$lib/common'
-    import { SearchPatternType } from '$lib/graphql-types'
+    import { SearchPatternType } from '$lib/graphql-operations'
     import Icon from '$lib/Icon.svelte'
-    import type { QueryState } from '$lib/search/state'
     import SyntaxHighlightedQuery from '$lib/search/SyntaxHighlightedQuery.svelte'
     import { buildSearchURLQuery } from '$lib/shared'
+    import { settings } from '$lib/stores'
     import TabPanel from '$lib/TabPanel.svelte'
     import Tabs from '$lib/Tabs.svelte'
+    import { showQueryExamplesForKeywordSearch } from '$lib/web'
     import ProductStatusBadge from '$lib/wildcard/ProductStatusBadge.svelte'
 
     import { basicSyntaxColumns, exampleQueryColumns, type QueryExample } from './queryExamples'
 
     export let showQueryPage = false
     export let queryExample: Readable<QueryExample | null>
-    export let queryState: QueryState
+
+    const queryExamplesForKeywordSearch = showQueryExamplesForKeywordSearch({ final: $settings, subjects: [] })
+    const patternTypeForQueryLinks = queryExamplesForKeywordSearch
```

Comment:
Yes, these two should be reactive:
```suggestion
    $: queryExamplesForKeywordSearch = showQueryExamplesForKeywordSearch({ final: $settings, subjects: [] })
    $: patternTypeForQueryLinks = queryExamplesForKeywordSearch
```

I agree with `Settings`/`SettingsCascade` part but we have that pattern already elsewhere int the code. I'll clean it up eventually.